docs/amy_database_structure.md

```diff
@@ -143,6 +143,7 @@ The primary tables used in AMY (that will likely appear in every query) are thos
 * `fullname` Human friendly name of the organization
 * `country` Stored as the [two digit country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
 * `latitude` and `longitude` Stored as floating point (decimal) numbers
+* `affiliated_organizations` Many-to-many relationship between organizations
```

Comment:
So https://github.com/carpentries/amy/issues/1836 says it's not the same as a consortium. Could you add a few lines on how this should be used?

Reply:
Sure, good idea.